package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "colette-kss-builder",
-  "version": "5.4.0",
+  "version": "5.5.0",
```

Comment:
Shouldn’t this be 5.4.1 instead?

Reply:
Right! I messed up with the npm cli and forgot to change here ^^
Good catch!